src/amber/cli.cr

```diff
@@ -4,4 +4,12 @@ require "./exceptions/*"
 require "./environment"
 require "./cli/commands"
 
+{% if flag? :amber_release %}
+  AMBER_SHARD_TARGET = :current_version
+{% elsif flag? :amber_stable %}
+  AMBER_SHARD_TARGET = :stable_branch
+{% else %}
+  AMBER_SHARD_TARGET = :master_branch
```

Comment:
Certainly there's a better place for these definitions, don't you think, @eliasjpr @elorest?

Reply:
We do have the `.amber.yml` file that stores CLI config settings.

Reply:
Please, see my comments here: https://github.com/amberframework/amber/pull/727#issuecomment-380900041